docs/docs/Contributing/contributing-how-to-contribute.md

```
@@ -112,4 +112,11 @@ Some additional guidance on pull request titles:
 * Pull request titles appear in Langflow's release notes, so they should explain what the PR does as explicitly as possible.
 * Pull requests should strive to fix one thing **only**, and should contain a good description of what is being fixed.
 
-For more information, see the [Python Developer's Guide](https://devguide.python.org/getting-started/pull-request-lifecycle/index.html#making-good-commits).
\ No newline at end of file
+For more information, see the [Python Developer's Guide](https://devguide.python.org/getting-started/pull-request-lifecycle/index.html#making-good-commits).
+
+## Additional contributing guides
+
+- [Contributing Bundles](./contributing-bundles.md)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)